swlor2_mdl/C_MK1DRD.mdl

```diff
@@ -0,0 +1,16577 @@
+# NeverBlender 4.0.0|Blender 4.0.2|2024-01-04
```

Comment:
Bunch of models in here need to be compiled

Reply:
Mdls compiled (dunno how I created a third empty commit)

Reply:
Still not compiled. See this one as an example: https://github.com/zunath/SWLOR_Haks/blob/4f1ae99195d991db1fbd82f46ef56be89effec06/swlor2_mdl/C_Drexl.mdl